src/test/java/studio/magemonkey/fabled/dynamic/condition/AirConditionTest.java

```diff
@@ -0,0 +1,63 @@
+package studio.magemonkey.fabled.dynamic.condition;
+
+import be.seeseemelk.mockbukkit.entity.PlayerMock;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.LivingEntity;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import studio.magemonkey.codex.mccore.config.parse.DataSection;
+import studio.magemonkey.fabled.testutil.MockedTest;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class AirConditionTest extends MockedTest {
+    private PlayerMock        player;
+    private LivingEntity      target;
+    private AirCondition condition;
+    private DataSection       settings = new DataSection();
+
+    @BeforeEach
+    public void setup() {
+        player = genPlayer("Travja");
+        target = (LivingEntity) player.getWorld()
+                .spawnEntity(player.getLocation(), EntityType.ZOMBIE);
+
+        DataSection data = new DataSection();
+        data.set("min-value-base", 20);
+        data.set("min-value-scale", 0);
+        data.set("max-value-base", 50);
+        data.set("max-value-scale", 0);
+        settings.set("data", data);
+
+        condition = new AirCondition();
+        condition.load(null, settings);
+    }
+
+    @Test
+    void test_outOfBounds() {
+        player.setRemainingAir(300);
```

Comment:
We probably don't need to set the player air values here because the target should be the one we're worried about.

Reply:
Removed, as it was not necessary.